### DISCUSSION TITLE
How to pin tabs so that they always show, even when scrolling down?

### DISCUSSION THREAD

**Discussion by jparthasarthy** [squidfunk/mkdocs-material#2100] [Q&A] [answered]
Tabs are my favorite feature. Love em. Love em so much I want them to always show even when people scroll down. Right now they auto collapse. Is there a way to do this? Thanks a bunch!

**Comment by SaltyAimbOtter** [comment]
Im pretty sure that this is not possible. I'm also a big fan of stopping auto collapse and have looked around the docs quite a few times in all the time I have been using this awesome theme. 
If you have the skills (unlike me 😓) to edit CSS / HTML / JS you can probably add it yourself as the theme is pretty [extensible](https://squidfunk.github.io/mkdocs-material/customization/). 

@squidfunk I would love to see this feature (in insiders ❤️). 😏

**Reply by squidfunk** [reply]
It's definitely on the roadmap for Insiders as this feature request has come up quite a few times, I just can't put a date on it yet 😊

**Comment by squidfunk** [comment, marked as answer]
Insiders 1.13.0 adds support for [sticky navigation tabs](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#sticky-navigation-tabs)! I re-prioritized the roadmap and as this feature was requested quite a few times, I thought it might be good to bump it up the list. Here's how it looks:

![Ohne Titel](https://user-images.githubusercontent.com/932156/102016974-29aa9080-3d64-11eb-87c6-accafcbee5bd.gif)

I'm marking this discussion as answered as this feature is now natively supported by Material for MkDocs Insiders.
